src/main.cpp

```diff
@@ -48,8 +49,14 @@ unsigned long loopTime = 0;
 unsigned long lastStatePrint = 0;
 bool secondImuActive = false;
 BatteryMonitor battery;
+TPSCounter tpsCounter;
 
 void setup() {
+	// For some reason, serial on C3 just doesn't work without the delay
+	// It can probably be less, but it needs investigation
+#ifdef ESP32C3
+	delay(3000);
+#endif
```

Comment:
This was removed in a different PR (https://github.com/SlimeVR/SlimeVR-Tracker-ESP/pull/329), I assume you mean the logs get cut off at the beginning? A ring buffer should be implemented for logs to solve this imo

Reply:
See discord: doesn't work for me. actually getting no serial if i don't put the delay, and i couldn't figure out why yet.